pkg/kubelet/types/constants.go

```diff
@@ -21,3 +21,21 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// HairpinMode denotes how the kubelet should configure networking to handle
+// hairpin packets.
+type HairpinMode string
+
+// Enum settings for different ways to handle hairpin packets.
+const (
+	// Set the hairpin flag on the veth of containers in the respective
+	// container runtime.
+	HairpinVeth = "hairpin-veth"
+	// Make the container bridge promiscuous. This will force it to accept
+	// hairpin packets, even if the flag isn't set on ports of the bridge.
+	HairpinBridge = "promiscuous-bridge"
```

Comment:
yeah, i meant variable name. I do prefer PromiscousBridge to be consistent with the flag string..
if you don't care, I would rename it. Also it is used as HairpinMode.PromiscousBridge, so its easy to guess whats going on...

I will LGTM the PR after you address this comment, either by renaming the variable, or saying you are not going to rename it :))